tests/backends/sqlite/tests.py

```diff
@@ -162,3 +162,27 @@ def create_object():
         thread.start()
         thread.join()
         self.assertEqual(Object.objects.count(), 2)
+
+
+@unittest.skipUnless(connection.vendor == 'sqlite', 'SQLite tests')
+class IntrospectionTests(TestCase):
+    def test_get_primary_key_column(self):
+        """
+        Validate that we can get the field name of the primary key.
```

Comment:
Docstrings should state the expected behavior and omit prefixes like "Validate that" since all tests validate things -- a guideline from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).

Reply:
Understood, thanks. I'll make the change.